### ISSUE TITLE
Improve user identifiers in initial Slack message

### ISSUE THREAD

**Issue by lmvdz** [xyfir/yalcs#2]
![image](https://user-images.githubusercontent.com/2179775/59152029-6294d700-8a02-11e9-91a7-f264258efca4.png)

Would it be possible to provide some configuration as to what to pass 

instead of ::ffff:127.0.0.1, could be something like: window meta title - ip geo location

**Comment by MrXyfir** [comment]
Yes, I'd like to improve the identifiers to include proper geolocation, window title, window URL, and also custom metadata (like username/email) that the web client could pass. As for now, it's just IP.

I'm assuming you have Nginx or something similar in front of the Node process that is losing the IP address?

**Comment by MrXyfir** [comment]
Geoip data is now included in `v1.2`. More data will be passed in later versions.
